arlo-client/src/components/AuditForms/EstimateSampleSize.tsx

```diff
@@ -193,13 +193,20 @@ const EstimateSampleSize: React.FC<IProps> = ({
     }
     try {
       setIsLoading(true)
-      await api(`/election/${electionId}/audit/basic`, {
-        method: 'POST',
-        body: JSON.stringify(data),
-        headers: {
-          'Content-Type': 'application/json',
-        },
-      })
+      const response: IErrorResponse = await api(
+        `/election/${electionId}/audit/basic`,
+        {
+          method: 'POST',
+          body: JSON.stringify(data),
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        }
+      )
+      if (toaster(response)) {
```

Comment:
I agree with @benadida on the naming of this. It's really not clear from the name `toaster` (and the fact it's in the `utilities` module) that this has a UI side effect since it looks here like it's just checking for something in the response.